models/Context.php

```diff
@@ -208,30 +208,32 @@ protected function updateSubInterfaceInheritance($interface)
     protected function inheritDocs($class)
     {
         // inherit for properties
-        foreach ($class->properties as $p) {
-            if ($p->hasTag('inheritdoc') && ($inheritTag = $p->getFirstTag('inheritdoc')) !== null) {
-                $inheritedProperty = $this->inheritPropertyRecursive($p, $class);
-                if (!$inheritedProperty) {
-                    $this->errors[] = [
-                        'line' => $p->startLine,
-                        'file' => $class->sourceFile,
-                        'message' => "Method {$p->name} has no parent to inherit from in {$class->name}.",
-                    ];
-                    continue;
-                }
+        if (!empty($class->properties)) {
```

Comment:
This one fixes the null set by default on interfaces here: https://github.com/yiisoft/yii2-apidoc/blob/master/models/InterfaceDoc.php#L44-L45

The empty check either needs to happen here or the property should be set to an empty array.

Reply:
https://github.com/yiisoft/yii2-apidoc/commit/1855afb6ad6810520925e3908994ef96ea06ae99